src/lib/core/say.c

```diff
@@ -651,6 +662,8 @@ log_syslog_init(struct log *log, const char *init_str)
 		log->syslog_facility = opts.facility;
 	say_free_syslog_opts(&opts);
 	log->fd = log_syslog_connect(log);
+	log->isatty = isatty(log->fd);
+	log->isatty_stderr = isatty(STDERR_FILENO);
```

Comment:
This chunk is not neede, since we have already set these fields in log_syslog_connect